web-animations-1/Overview.bs

```diff
@@ -731,6 +750,19 @@ A <a>document timeline</a> that is associated with a {{Document}} which is not
 an <a>active document</a> is also considered to be
 <a lt="inactive timeline">inactive</a>.
 
+The [=timeline phase|phase=] of a <a>document timeline</a> at a given moment is
+the [=timeline phase|phase=] corresponding to the <em>first</em> matching
+condition from the following:
+
+<div class="switch">
+
+:   If the <a>document timeline</a> is <a lt="inactive timeline">inactive</a>:
+::  &rarr; [=timeline inactive phase|inactive=]
+:   Otherwise,
+::  &rarr; [=timeline active phase|active=]
```

Comment:
I'm probably missing something but this seems like a circular reference? Can't we just say that it's active except for in such-and-such a situation where it's inactive?

Reply:
It's not quite circular but I understand the confusion. "Inactive" is defined differently for each type of timeline. "Inactive" is defined for DocumentTimeline above:

"Prior to establishing the time origin for its associated document, a document timeline is inactive."
"A document timeline that is associated with a document which is not an active document is also considered to be inactive"

So depending on the document, timeline can be inactive, and based on whether or not the timeline is active we set the phase.

A lot of that was existing, should I change it around to make it less confusing?